mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -37,6 +37,42 @@ import org.apache.spark.sql.types._
  */
 private[feature] trait RFormulaBase extends HasFeaturesCol with HasLabelCol {
 
+  /**
+   * Param for how to order categories of a string FEATURE column used by `StringIndexer`.
+   * The last category after ordering is dropped when encoding strings.
+   * Supported options: 'frequencyDesc', 'frequencyAsc', 'alphabetDesc', 'alphabetAsc'.
+   * The default value is 'frequencyDesc'. When the ordering is set to 'alphabetDesc', `RFormula`
+   * drops the same category as R when encoding strings.
+   *
+   * The options are explained using an example `'b', 'a', 'b', 'a', 'c', 'b'`:
+   * {{{
+   * +-----------------+---------------------------------------+----------------------------------+
```

Comment:
Ah, sure, I initially meant a HTML list that we are already using - https://github.com/apache/spark/blob/04901dd03a3f8062fd39ea38d585935ff71a9248/sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala#L304-L340 ... 

```html
<ul>
  <li> abc </li>
  <li> abc </li>
</ul>
```

I just tested it to double-check a wiki-style list ( `-` ) - http://subnormalnumbers.blogspot.kr/2011/08/scaladoc-wiki-syntax.html. This does not work correctly as below (but please go ahead if you know any compatible way for both Scaladoc and Javadoc):

```
   *  1. item one
   *
   *  1. item two
   *    - sublist
   *    - next item
   *
   *  1. now for broken sub-numbered list, the leading item must be one of
   *     `-`, `1.`, `I.`, `i.`, `A.`, or `a.`. And it must be followed by a space.
   *    1. one
   *    2. two
   *    3. three
   *
   *  1. list types
   *    I. one
   *      i. one
   *      i. two
   *    I. two
   *      A. one
   *      A. two
   *    I. three
   *      a. one
   *      a. two
```

Scaladoc

![2017-05-23 9 52 51](https://cloud.githubusercontent.com/assets/6477701/26334123/b153378a-3f9d-11e7-8852-31b519ec9f21.png)

Javadoc

![2017-05-23 9 53 07](https://cloud.githubusercontent.com/assets/6477701/26334121/af2696d2-3f9d-11e7-9924-f0e4373b2cce.png)



My worry is, it draws attention with a different format. I believe we have similar instances but wonder if it is worth changing only this one. I would not strongly against but `{{{ ... }}}` basically means codes. If we can't find a better way to render this, I would leave this out as prose with a list.